detect_secrets/custom_types.py

```diff
@@ -1,7 +1,6 @@
 from io import TextIOBase
 from typing import Any
 from typing import NamedTuple
-from typing import NoReturn
```

Comment:
can you leave this and remove the code in lines 11-28, it is the try/except block with `from typing import NoReturn`

Reply:
changed it :)